### DISCUSSION TITLE
Support for QtHelpProject?

### DISCUSSION THREAD

**Discussion by LazyT** [WeblateOrg/weblate#5823] [Ideas]
Hi,

I'm new to Weblate and setup my first project with *.ts and *.json as sources, which works fine so far.

But [QtHelp](https://doc.qt.io/qt-5/qthelpproject.html#qt-help-project-file-format) has also an xml (named *.qhp), which is rejected by Weblate's XML file type.

How can I translate these files?

Or is there any chance to get this added in future?

It needs `<section title="translatable title string" .../>` fields and optional `<keyword name="translatable keyword string" .../>`.

Thanks!

**Comment by nijel** [comment]
We don't have any plans for this format so far. See https://docs.weblate.org/en/latest/formats.html#supporting-other-formats for generic info.
